commit 6bba71df4b06b3ee1f6c1d7d6fbd552dc655ed0b (refs/changes/42/3942/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-05-11T15:21:11-07:00 (1 year, 5 months ago)
    
    MANTA-3684 manta-scripts needs to work around default logadm.conf in new base image

diff --git a/deps/manta-scripts b/deps/manta-scripts
index dc593cd..cd8d6f1 160000
--- a/deps/manta-scripts
+++ b/deps/manta-scripts
@@ -1 +1 @@
-Subproject commit dc593cd90475e391c6fce874130f5d522dc0cc54
+Subproject commit cd8d6f194ba34dfaa4b0740dfbd11147041dd13e
